content/04.network_applications.md

```diff
@@ -1,16 +1,16 @@
 ## Applying Knowledge Graphs to Biomedical Challenges
 
-Knowledge graphs can help researchers tackle many biomedical tasks such as finding new treatments for existing drugs [@doi:10.7554/eLife.26726], aiding efforts to diagnose patients [@arxiv:1611.07012] and predicting associations between diseases and biomolecules [@doi:10.1155/2017/2498957].
-In many cases, solutions rely on representing knowledges graphs in a low dimensional space, which is a process called representation learning.
-This space preserves a knowledge graph's local and/or global structure and can support efforts to apply machine learning methods to make predictions.
-In the next sections we review the unifying techniques that construct this low dimensional space and unifying applications that use this space to solve biomedical problems.
+Knowledge graphs can help researchers tackle many biomedical problems such as finding new treatments for existing drugs [@doi:10.7554/eLife.26726], aiding efforts to diagnose patients [@arxiv:1611.07012] and identfying associations between diseases and biomolecules [@doi:10.1155/2017/2498957].
+In many cases, solutions rely on representing knowledge graphs in a low dimensional space, which is a process called representation learning.
+This space preserves a knowledge graph's local and/or global structure and is straightforward to utilize in efforts that apply machine learning methods to make predictions.
```

Comment:
```suggestion
This goal of this process is to retain and encode the local and/or global structure of a knowledge graph that are relevant to the problem while transforming the graph into a representation that can be readily used with machine learning methods to build predictors
```
Maybe this will help with the reviewer?